### ISSUE TITLE
Whenever I'm creating a new interface class in Android I got this error

### ISSUE THREAD

**Issue by zeeshan5446** [ignatov/intellij-erlang#809]
Unable to parse template "Interface"

    Error message: This template did not produce a Java class or an interface

**Comment by domgeorg** [comment]
File > Settings > Editor > File and Code Templates 
Select Java Interface from Files Tab and paste this: `#if (${PACKAGE_NAME} && ${PACKAGE_NAME} != "")package ${PACKAGE_NAME};#end #parse("File Header.java") public interface ${NAME}{ }`

**Comment by zys5945** [comment]
That indeed solved the problem. However, whenever I close and restart android studio I get that error again... Is there a way to perma fix this?

**Comment by MusaddiqueAli** [comment]
Yes, even I am facing this issue with the Android Studio, is there a permanent solution to this?

**Comment by rajeshfsp** [comment]
if anyone knows please update the answer. I'm using AS 3.0.1
